### DISCUSSION TITLE
How to enable Italic font

### DISCUSSION THREAD

**Discussion by mxCynic** [ghostty-org/ghostty#3360] [Q&A]
This is my ghostty conf cat at gostty
![758AB44327DAEAF29FA7658CDB8304B0](https://github.com/user-attachments/assets/619b3371-7e86-46a1-85cf-f6768824d41c)
It's like my kitty conf 
![image](https://github.com/user-attachments/assets/ec58f715-b8e1-436e-a4c6-81368cf28a5b)
I dont know why ghostty is not italic

**Comment by ciscohack** [comment]
I have same question.. Any expert help how to enable Italic style for text rendering in Ghostty. in Kitty terminal config if i mentioned liked this it renderers italic style text but in Ghostty it doesn't recognize the font with this way why?

<img width="213" alt="image" src="https://github.com/user-attachments/assets/99561733-89d0-4efe-b759-4b01bf5d5156" />

**Reply by bjd-pfq** [reply]
I have the same issue.
I have `font-family = Fantasque Sans Mono Italic` to have all 'regular' text in italics, but ghostty does not seem to want to do it.

**Reply by rhodes-b** [reply]
I believe `font-family` should be the base font name and not a specific derivative (I could be wrong on that)
Then you would set the font style or disable the styles you dont want like 
https://ghostty.org/docs/config/reference#font-style
options state

**Reply by 00-kat** [reply]
Hold on.  @bjd-pfq you want to set [`font-style-italic`].  But this post seems to be that setting `font-style` to an italic font doesn't work (i.e., OP *wants* everything to be italic).  Are you sure you have the same issue?  Or perhaps I completely misinterpreted this post.

[`font-style-italic`]: https://ghostty.org/docs/config/reference#font-style-italic
